src/fermilib/utils/_operator_utils.py

```diff
@@ -153,8 +153,8 @@ def save_operator(operator, file_name=None, data_directory=None):
 
     tm = operator.terms
     with open(file_path, 'wb') as f:
-        marshal.dump((operator_type, dict(izip(tm.iterkeys(),
-                                          imap(complex, tm.itervalues())))), f)
+        marshal.dump((operator_type, dict(zip(tm.keys(),
```

Comment:
Should be fine as is! If it's significantly slower I'll make another PR after this one gets through, but I doubt there'll be any issues.